src/main/java/io/github/opendme/server/entity/Member.java

```diff
@@ -19,12 +22,11 @@ public class Member {
     @Id
     @GeneratedValue
     private Long id;
-    @ManyToOne
+    @ManyToOne(cascade = CascadeType.REMOVE)
     private Department department;
     private String name;
-    @ManyToMany
-    @JoinTable
-    private Set<Skill> skills;
+    @OneToMany(mappedBy = "member", cascade = CascadeType.REMOVE)
+    private Set<MemberSkill> skills;
```

Comment:
is that correct? 
`OneToMany` sounds to me like one member has many skills and many skills just have one member. 
I would think that many skills have many member, because a skill like "Driver" is used by many members.

Reply:
Oh i just saw the new `MemberSkill` entity.

Reply:
Thats why its a member skill now. Its only a link between a member and a skill. Thats why one member has now many links, but not many links have many members.
I basically just modeled the join table we had before with an explicit entity.